### PR TITLE
add create order view

### PR DESCRIPTION
**Pull request by Isterius**

### CODE REVIEW

File: apps/frontend/src/app/CreateOrder/createOrder.model.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export type CreateOrderPayload = Readonly<{
```
Comment (pawelkondraciuk) [inline apps/frontend/src/app/CreateOrder/createOrder.model.ts]: Move it to libs/api/data-transfer/src/lib/create-order.dto.ts

Comment (pawelkondraciuk) [reply apps/frontend/src/app/CreateOrder/createOrder.model.ts]: Or even use it directly from there rather than extra models

File: package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "nx": "nx",
     "start": "nx serve",
     "start:api": "nx serve nestjs-api",
-    "start:database": "npm run docker-compose up",
+    "start:database": "npm run docker-compose up -- -d",
```
Comment (pawelkondraciuk) [inline package.json:9]: I left it as non detached to easily stop the database

Comment (pawelkondraciuk) [reply package.json:9]: You could add extra command to run db detached

**Review by pawelkondraciuk — COMMENTED** [review]
What do you think about making shippingCents 0 as default? It could be changed on mark as ordered and mark as delivered state

**Comment by Isterius** [comment]
@pawelkondraciuk sure 👍 
also - I consider `from` to be required field 🤔 let's force it in create order request 👀

**Review by pawelkondraciuk — APPROVED** [review]

**Pull request merged**